bitnami/mariadb/templates/update-password/job.yaml

```diff
@@ -0,0 +1,241 @@
+{{- /*
+Copyright Broadcom, Inc. All Rights Reserved.
+SPDX-License-Identifier: APACHE-2.0
+*/}}
+
+{{- if .Values.passwordUpdateJob.enabled }}
+apiVersion: batch/v1
```

Comment:
Don't we have a `common.capabilities.XXX.apiVersion` for this?

Reply:
We don't. We have it from cronjobs but not for jobs